STYLE_GUIDE.md

```diff
@@ -93,10 +93,10 @@ It's worth noting that GitHub Markdown only cares which numbers you use for the
 it takes that as the initial value and counts up from there.
 As such, there are two methods that work:
 
-1.  Usually it's preferable to number all items with `1`.
-    This keeps diffs clean since inserting or removing an item doesn't re-number all subsequent items.
-1.  However, if the doc refers to the items _by number_ it makes sense to number them ordinarily within the Markdown.
-    The Prettier will _correct the order of items_ numbered in this way which is nice (but does cause noisier diffs).
+1. Usually it's preferable to number all items with `1`.
+   This keeps diffs clean since inserting or removing an item doesn't re-number all subsequent items.
+2. However, if the doc refers to the items _by number_ it makes sense to number them ordinarily within the Markdown.
+   The Prettier will _correct the order of items_ numbered in this way which is nice (but does cause noisier diffs).
```

Comment:
Reading through this style guide, I noticed a lot of these rules can be enforced through [`remark-lint`](https://github.com/remarkjs/remark-lint). I hadn't planned to put a PR together for this, as it could end up being quite large..

By default `remark` will just behave as a "pretty printer". To enable linting (or any additional behaviour) a plugin has to be introduced.